src/routes/docs/tutorials/nextjs/step-4/+page.markdoc

```diff
@@ -0,0 +1,218 @@
+---
+layout: tutorial
+title: Implement authtentication
+description: Add Appwrite authentication to your Next.js app
+step: 4
+---
+
+# Signup page {% #signup-page %}      
```

Comment:
<img width="860" alt="Screenshot 2023-10-13 at 3 59 55 PM" src="https://github.com/appwrite/website/assets/29069505/234c1233-1490-449c-8ea6-0c06e7d017ff">

Reply:
Can you give me more details on this? I have added cookie import both login and signup component